### ISSUE TITLE
14.22 micropatch not working

### ISSUE THREAD

**Issue by evelynnie** [hydy100/R3nzSkin#218]
Hi, I know you said earlier in another issue thread that the injector still works after the micropatch but I'm having issues.
It's not having the regular error code that comes after most patches but it isn't showing up in game, it says it's active but there's just no injection

any advice on how to fix?

**Comment by evelynnie** [comment]
i dont think you understand, i meant the error message usually comes through when there is an update. there is not an error message this time, it just doesnt show up in game.

**Comment by rangigo** [comment]
@hydy100 can you check again please? For me 14.22 was working last Sunday (10.11). But on Monday I tried to use the injector, pressing the home button and then in game pressing insert the menu does not show up

**Comment by DK1192** [comment]
Not working for me also

**Comment by Zaya619** [comment]
for me, my antivirus started acusing this as a trojan and is blocking it, and that's why is not working, i tried adding exclusive access and all, but nope, doesn't work, my antivirus just doesn't let the file be loaded by anymeans... any thoughts on why?

**Comment by rangigo** [comment]
It is working now. Thanks for the quick update hydy :)
